Validate mapper (#1146)
demisto/demisto-sdk: CHANGELOG.md

* Fixed an issue where **validate** would incorrectly identify files as renamed.
* Added a validation that integration properties (such as feed, mappers, mirroring, etc) are not removed.
* Fixed an issue where **validate** failed when comparing branch against commit hash.

> * Added a validation that there's no change is incidents fields, or removal of incidents types

Reviewer: ```suggestion
* Added a validation that incident fields and incident types are not removed from mappers.
```
